Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG ARCH=
+FROM ${ARCH}rust:1.73 as builder
+RUN apt-get update && \
+  apt-get install --no-install-recommends -y \
+  ca-certificates \
+  && rm -rf /var/lib/apt/lists/*
+WORKDIR /app
+COPY . .
+WORKDIR /app/crates/bws
+RUN cargo build --release
+FROM ${ARCH}debian:trixie-slim as runner
```

Comment:
This is the pre-release debian image, which seems like an odd choice.

Reply:
So, I tried using `bookworm-slim` before, but I ran into an SSL issue. However, after re-testing just now, I was able to auth and list secrets. I'll get this switched to `bookworm-slim`.